client/repositories/operation/operation.go

```diff
@@ -0,0 +1,193 @@
+package operation_repo
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/lidofinance/dc4bc/client/modules/state"
+	"github.com/lidofinance/dc4bc/client/types"
+)
+
+const (
+	OperationsKey        = "operations"
+	DeletedOperationsKey = "deleted_operations"
+	FSMStateKey          = "fsm_state"
```

Comment:
FSMStateKey is another constant defined in several places, and this time it's defined in three files: state, operation repo and fsm service. I think we need to stick with a single source of truth, do you agree? i.e. it's a state package related constant, doesn't it belong there then? if so, operation repo and fsm service should import it from there